guides/micronaut-openapi-generator-client/java/src/main/resources/application.yml

```diff
@@ -0,0 +1,30 @@
+base-path: "https://api.twitter.com/"
+context-path: "/"
+
+# tag::authorization[]
+micronaut:
+  application:
+    name: openapi-micronaut
+  server:
+    port: 8080
+  security:
+    oauth2:
+      clients:
+        twitter: # <1>
+          client-id: '${TWITTER_AUTH_CLIENT_ID}' # <4>
+          client-secret: '${TWITTER_AUTH_CLIENT_SECRET}'
+          client-credentials:
+            service-id-regex: '.*' # <5>
```

Comment:
I think you should only decorate the request going to `https://api.twitter.com`. I think you can do it with `
`micronaut.security.oauth2.clients.*.client-credentials.uri-pattern` setting.